requirements/edx/stanford.txt

```diff
@@ -0,0 +1,10 @@
+xblock-image-modal >=0.2.0, <1.0.0
+-e git+https://github.com/openlearninginitiative/xblock-image-coding.git@cee23e3d59943066159ca13839c82a12d9ccf749#egg=xblock_image_coding
+-e git+https://github.com/Stanford-Online/xblock-qualtrics-survey@653400485196f8bb05b859c9d89de378a22e0503#egg=xblock-qualtrics-survey
+-e git+https://github.com/Stanford-Online/xblock-free-text-response@01cb026337f4fdd97e89b017142be06506e56afa#egg=xblock-free-text-response
+-e git+https://github.com/Stanford-Online/xblock-grademe.git@v0.1.0#egg=grademebutton
+-e git+https://github.com/edx-solutions/xblock-ooyala.git@32d52edaa820dbdbf846d8a84f6bccbf0b9c8218#egg=xblock_ooyala_player-master
```

Comment:
I'm a little confused as to the rationale for what goes where in this scheme. For example, the ooyala xblock is a third party xblock, not Stanford's.